app/src/main/res/layout-land/profile_chooser_fragment.xml

```diff
@@ -39,30 +39,24 @@
           app:layout_constraintTop_toTopOf="parent" />
 
         <TextView
-          android:layout_width="wrap_content"
-          android:layout_height="wrap_content"
-          android:fontFamily="sans-serif"
+          style="@style/Subtitle1"
```

Comment:
Reformat the code.
Press `Control + Alt + L` - reformatting shortcut to fix the formatting.

Reply:
Do this in all files.

Reply:
Done